src/libs/actions/IOU.js

```diff
@@ -75,6 +77,30 @@ function requestMoney(report, amount, currency, recipientEmail, participant, com
         iouReport = ReportUtils.buildOptimisticIOUReport(recipientEmail, debtorEmail, amount, chatReport.reportID, currency, preferredLocale);
     }
 
+    const optimisticTransaction = TransactionUtils.buildOptimisticTransaction(amount * 100, currency);
+    const optimisticTransactionData = {
+        onyxMethod: Onyx.METHOD.MERGE,
```

Comment:
In the past, we had a UI delay when using `MERGE` to first insert data in Onyx and we had to change to `SET` to fix the issue, like we did [here](https://github.com/Expensify/App/blob/30420a19d14bbc021189d9bbfe7b702fcd1c2323/src/libs/actions/IOU.js#L154-L155). Just something to keep in mind when testing.